mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMessageIdManager.java

```diff
@@ -239,7 +239,7 @@ private MessageMetaData save(MailboxSession mailboxSession, MessageIdMapper mess
         MessageUid uid = mailboxSessionMapperFactory.getUidProvider().nextUid(mailboxSession, mailboxMessage.getMailboxId());
         mailboxMessage.setModSeq(modSeq);
         mailboxMessage.setUid(uid);
-        messageIdMapper.save(mailboxMessage);
+        messageIdMapper.saveInMailbox(mailboxMessage);
```

Comment:
we keep both methods ? How am I supposed to choose which one to call ? I don't understand the difference when looking at function names